scripts/check_driver.sh

```diff
@@ -0,0 +1,230 @@
+                else
+                    echo -e "❌ \e[91mPilot not found.\e[0m\n"
+                fi
+                return
+
+            fi
+            ;;
+        "Solus Budgie")
+            print_distro $DISTRO
+            if ! type "vaino" > /dev/null 2>&1
+            then
+                print_vainfo_warning
+                sudo eopkg it libva-utils
+            else
+                echo -e "✅ \e[32mFound vainfo.\e[0m\n"
+
+                if [[ check_h264 ]] && [[ check_h265_hevc ]]
+                then
+                    echo -e "✅ \e[32mH264 found.\e[0m"
+                    echo -e "✅ \e[32mH265 / HEVC found.\e[0m\n"
+                    return
+                fi
+                if [[ ! check_h264 ]]
+                then
+                    echo -e "❌ \e[91mH264 not found.\e[0m\nPlease generate a report with the following command:\n\t- curl https://raw.githubusercontent.com/NicolasGuilloux/blade-shadow-beta/master/scripts/report.pl | perl"
+                fi
+                if [[ ! check_h265_hevc ]]
+                then
+                    echo -e "❌ \e[91mH265 or HEVC not found.\e[0m"
+                fi
+
+                pilot=$(vainfo |& grep 'vainfo: Driver version:')
+                print_driver_warning
+                if [[ $pilot == *"Intel"* ]]
+                then
+                    sudo eopkg it libva-intel-driver
+                elif [[ $pilot == *"NVIDIA"* ]]
+                then
+                    print_nvidia
+                else
+                    echo -e "❌ \e[91mPilot not found.\e[0m\n"
+                fi
+                return
+
+            fi
+            ;;
+        "Fedora")
+            print_distro $DISTRO
+            if ! type "vaino" > /dev/null 2>&1
+            then
+                print_vainfo_warning
+                sudo dnf install librtmp
+            else
+                echo -e "✅ \e[32mFound vainfo.\e[0m\n"
+
+                if [[ check_h264 ]] && [[ check_h265_hevc ]]
+                then
+                    echo -e "✅ \e[32mH264 found.\e[0m"
+                    echo -e "✅ \e[32mH265 / HEVC found.\e[0m\n"
+                    return
+                fi
+                if [[ ! check_h264 ]]
+                then
+                    echo -e "❌ \e[91mH264 not found.\e[0m\nPlease generate a report with the following command:\n\t- curl https://raw.githubusercontent.com/NicolasGuilloux/blade-shadow-beta/master/scripts/report.pl | perl"
+                fi
+                if [[ ! check_h265_hevc ]]
+                then
+                    echo -e "❌ \e[91mH265 or HEVC not found.\e[0m"
+                fi
+
+                pilot=$(vainfo |& grep 'vainfo: Driver version:')
+                print_driver_warning
+                if [[ $pilot == *"Intel"* ]]
+                then
+                    sudo dnf install libva-intel-hybrid-driver libvdpau-va-gl
+                    sudo sh -c "echo 'export LIBVA_DRIVER_NAME=i965' >> /etc/profile"
```

Comment:
#55

Reply:
I disagree: all Intel GPU does not support the `i965`, and the `iHD` driver may be better as it should give better performance.

Reply:
reverted https://github.com/aquadzn/blade-shadow-beta/commit/945c0f8bf0ef034dcdf4df102163e4f42819b931